docs/Options.md

```diff
@@ -463,7 +463,9 @@ Ember.js section is shown only in directories that contain a `ember-cli-build.js
 
 ### Kubectl context (`kubecontext`)
 
-Shows the active kubectl context.
+Shows the active kubectl context, which consists of a cluster name and, when working in a non-default namespace, also a namespace name.
+
+TIP: If your cluster name (and thus context name) is too long, you can give it a shorter name using `kubectl config rename-context very_long_context_name name`.
```

Comment:
May be a bit highlighting to "tip" with bold or italics ?

Reply:
And write in `Tip` not `TIP`. Actually, in _npm_ section we use write it like that: https://github.com/denysdovhan/spaceship-prompt#npm

Reply:
You got it 😉